### PR TITLE
[Improve] code format and translate comments to EN.

### LINKED ISSUE #1522: [Feature] Code format，translate ZH to EN
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Code format，translate ZH to EN

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by lvshaokang**
Problem Summary: code format and translate comments to EN.

### What is changed and how it works?

Feel free to ping committers for the review!

## Contribution Checklist

  - Make sure that the pull request corresponds to a [GITHUB issue](https://github.com/apache/streampark/issues).

  - Name the pull request in the form "[Feature] [component] Title of the pull request", where *Feature* can be replaced by `Hotfix`, `Bug`, etc.

  - Minor fixes should be named following this pattern: `[hotfix] [docs] Fix typo in README.md doc`.

### CODE REVIEW

**Comment by lvshaokang** [comment]
@wolfboys PTAL.Thanks

**Comment by MonsterChenzhuo** [comment]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -622,8 +633,21 @@
                         <inputEncoding>UTF-8</inputEncoding>
                         <outputEncoding>UTF-8</outputEncoding>
                     </configuration>
+                    <dependencies>
```
Comment (wolfboys) [inline pom.xml:632]: Can you tell me what is the purpose of this dependency and why should it be added?

Comment (lvshaokang) [reply pom.xml:632]: @wolfboys Upgrade checkstyle version does not affect the normal build process, but if we not, execute mvn build will print warning like this: ```Old version of checkstyle detected. Consider updating to >= v8.30```

**Review by wolfboys — APPROVED** [review]
Approve this pull request

**Pull request merged**